src/server/interfaces.ts

```diff
@@ -20,6 +20,7 @@ export interface IRunnerHandlebarsContext {
 
     origin: string;
     host: string;
+    assets: any;
```

Comment:
>ssets: any; [](start = 5, length = 11)

Is there not a better type than any?  For example, it sounds like it's just a key-value pair JSON object.  If so, maybe declare it as `{ [key: string]: string }`?  Ditto inside the `server.ts` code as well.